awx/ui_next/src/screens/ExecutionEnvironment/shared/ExecutionEnvironmentForm.jsx

```diff
@@ -1,18 +1,41 @@
-import React from 'react';
+import React, { useCallback } from 'react';
 import { func, shape } from 'prop-types';
-import { Formik, useField } from 'formik';
+import { Formik, useField, useFormikContext } from 'formik';
 import { withI18n } from '@lingui/react';
 import { t } from '@lingui/macro';
-
 import { Form } from '@patternfly/react-core';
-import FormField, { FormSubmitError } from '../../../components/FormField';
-import FormActionGroup from '../../../components/FormActionGroup';
+
 import CredentialLookup from '../../../components/Lookup/CredentialLookup';
-import { url } from '../../../util/validators';
+import FormActionGroup from '../../../components/FormActionGroup';
+import FormField, { FormSubmitError } from '../../../components/FormField';
 import { FormColumnLayout } from '../../../components/FormLayout';
+import { OrganizationLookup } from '../../../components/Lookup';
+import { required, url } from '../../../util/validators';
+
+function ExecutionEnvironmentFormFields({ i18n, me, executionEnvironment }) {
+  const [credentialField, ,] = useField('credential');
+  const [organizationField, organizationMeta, organizationHelpers] = useField({
+    name: 'organization',
+    validate: !me?.is_superuser
+      ? required(i18n._(t`Select a value for this field`), i18n)
+      : () => undefined,
```

Comment:
```suggestion
validate: !me.is_superuser && required(i18n._(t`Select a value for this field`), i18n)
```
doing that removes the need for the ternary operator